container/dev/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+#
+# Run from the command line like e.g.: sh run.sh "/path/to/project/folder/"
+#
+
+
+if [ $# -eq 0 ]; then
+  echo "Error: No host project folder argument supplied." >&2
+  exit 1
+fi
+
+PROJECT_FOLDER=$1
+
+CONTAINER_NAME="dev-container"
```

Comment:
Please rename to `ppdev`:
- there may be other "dev" containers/process (like GFF), important the container name is associate with `pp`
- `-container` in the name is unnecessary, for sure we are dealing with a docker container here

Reply:
Changed.